tests/cocotb/snax_util.py

```diff
@@ -150,6 +150,8 @@ def extract_tcdm_list() -> Tuple[List[str], List[str]]:
 def gen_rand_int_list(list_len: int, min_val: int, max_val: int) -> List[int]:
     uint_list = []
 
+    random.seed(0)
```

Comment:
It got me thinking whether it's better to set the seed fixed to **0** OR to report the seed in the log. Like we randomize the seed, and report it in the log.

Fixing it to 0 is OK but we might miss a few cases that we may not know.